src/crimson/os/alienstore/alien_store.cc

```diff
@@ -465,6 +465,31 @@ auto AlienStore::omap_get_values(CollectionRef ch,
   });
 }
 
+auto AlienStore::omap_iterate(CollectionRef ch,
```

Comment:
nit: IMO, keeping explicit return types should help with reading / understanding the function

Reply:
done